Accounts/accountsRouter.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+
+const db = require("./../data/dbConfig");
+
+const router = express();
+
+router.get("/", (req, res) => {
+  db.select("*")
```

Comment:
you could shorten this, and not have to write db.select every time if you set a variable to the knexfile.
then this way you could simply write the table you want to use every time, just like how you were doing previously when you didn't have to write out the model logic in the router file